src/DynamoCoreWpf/ViewModels/Watch3D/DynamoPointLineRenderCore.cs

```diff
@@ -1,6 +1,10 @@
-﻿using System.Windows;
+using System.Windows;
 using HelixToolkit.Wpf.SharpDX;
```

Comment:
This needs to be moved to the `#else` block too?

Reply:
It is a common using that implements GeometryModel3D